src/main/kotlin/models/config/ConfigOptions.kt

```diff
@@ -20,6 +20,7 @@ class ConfigOptions(private val configuration: Configuration) {
     val alphaMode = ConfigOption("alpha-mode", ConfigOptionType.Enumerated(AlphaMode::valueOf, AlphaMode.values(), AlphaMode::humanReadableName), AlphaMode.ORDERED_DITHER, "Alpha mode", 'L')
     val sampleShading = ConfigOption("sample-shading", ConfigOptionType.boolean, false, "Sub-sample shading (GL4.0; makes hashed alpha look nicer)", 'S')
     val fov = ConfigOption("fov", ConfigOptionType.double, 90.0, "Field of view (degrees)", 'V')
+    val skyColor = ConfigOption("sky-color", ConfigOptionType.string, "#90DBE8", "Sky color (hex)", 'C')
```

Comment:
Instead of representing a String type, you could use a colour (`java.awt.Color`) which would allow you to use [`JColorChooser.showDialog`](https://docs.oracle.com/javase/8/docs/api/javax/swing/JColorChooser.html#showDialog-java.awt.Component-java.lang.String-java.awt.Color-) or (if you are comfortable handling things a little more manually) [`JColorChooser.createDialog`](https://docs.oracle.com/javase/8/docs/api/javax/swing/JColorChooser.html#createDialog-java.awt.Component-java.lang.String-boolean-javax.swing.JColorChooser-java.awt.event.ActionListener-java.awt.event.ActionListener-) when editing the value.

Since this would let us use a `Color` object as an internal representation, it would also remove the need to create the `Color` in the renderer object, and instead we can just grab the `Color` directly from config.

If you end up using `createDialog` (remembering to set the window to dispose on close), you might want to subclass the colour picker to ensure all the tabs are kept in sync:
```kotlin
class UpdatingColorChooser : JColorChooser() {
    init {
        // Set all colours in all choosers to the same value when one is changed
        selectionModel.addChangeListener { event ->
            for (chooser in chooserPanels) {
                if (chooser != event.source) {
                    chooser.updateChooser()
                }
            }
        }
    }
}
```

If you really wanted to, you could take the idea even further by also updating the corresponding configOption in the selection model listener there, which would allow a non-modal colour picker dialog to update the colour in the preview window in real time.